### PR TITLE
Use FQDN in pyvast-threatbus metrics

### PR DESCRIPTION
**Pull request by satta**
###  :notebook_with_decorative_cover: Description

This PR changes the metrics submitted by `pyvast-threatbus` to use the FQDN of the submitting system instead of the short hostname.

###  :memo: Checklist

- [X] All user-facing changes have changelog entries.
- [X] The changes are reflected on [docs.tenzir.com/threatbus](https://docs.tenzir.com/threatbus), if necessary.
- [X] The PR description contains instructions for the reviewer, if necessary.

### :dart: Review Instructions

Confirm that this change makes sense and is not overly specific.

### CODE REVIEW

**Review by mavam — APPROVED** [review]
Seems like a strict superset of the current functionality. If users would want the old behavior, they can still split the FQN.

@lava if you have no objections, feel free to merge.

**Review by 0snap — APPROVED** [review]
Green light from my side, too. What's blocking the merge?

**Comment by mavam** [comment]
> What's blocking the merge?

Nothing, just a second pair of eyes. 👀

**Comment by satta** [comment]
Thanks!

**Pull request merged**
